app/main.py

```
@@ -0,0 +1,55 @@
+# Get a single ToDo item
+@app.get("/todos/{todo_id}")
+async def read_todo(
+    todo_id: int, db: AsyncSession = Depends(get_db)
+) -> schemas.ToDoResponse:
+    return await crud.get_todo(db, todo_id)  # Error handling done in crud.get_todo
+
+
+# Create a new ToDo item
+@app.post("/todos", status_code=201)
+async def create_todo(
+    todo: schemas.ToDoCreate, db: AsyncSession = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)